webhookrelay/resource_webhookrelay_bucket.go

```diff
@@ -112,10 +120,12 @@ func resourceWebhookrelayBucketRead(d *schema.ResourceData, meta interface{}) er
 func resourceWebhookrelayBucketUpdate(d *schema.ResourceData, meta interface{}) error {
 	client := meta.(*client.Openapi)
 
-	if d.HasChange("name") || d.HasChange("description") {
+	if d.HasChange("name") || d.HasChange("description") || d.HasChange("websocket_streaming") || d.HasChange("ephemeral_webhooks") {
```

Comment:
Let's update this to [`d.HasChanges(...)`](https://github.com/hashicorp/terraform-plugin-sdk/pull/241)

Reply:
done